### PR TITLE
fix:1303 - Included dialog-polyfill for the support in firefox browser and updated to lit

### PR DESCRIPTION
**Pull request by riginoommen**
# Resolves 1303

# Explain the fix

Included the polyfills for the support in firefox for this component and updated to lit

## Does this PR introduce a breaking change
No

## Screenshots

<details>
<summary>View Screenshots</summary>

image

image

image

</details>

### Ready-for-merge Checklist

- [X] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [X] My changes generate no new warnings
- [X] I have performed a self-review of my own code

#### Browsers you have tested in
- [x] Google Chrome [Latest 2 versions]
- [x] Mozilla Firefox [Latest 2 versions]

### CODE REVIEW

**Review by hybridx — APPROVED** [review]
LGTM! @gisumit this would need the update for lit 2.0 as well

File: packages/opc-feedback/src/opc-feedback.ts [hunk 1]
```diff
@@ -11,12 +11,12 @@
  * Created at     : 2021-01-18 14:53:24 
  * Last modified  : 2021-03-04 21:38:14
  */
-
 import { LitElement, html, property, customElement, internalProperty, query } from 'lit-element';
 import style from './opc-feedback.scss';
 import { repeat } from 'lit-html/directives/repeat.js';
```
Comment (gisumit) [inline packages/opc-feedback/src/opc-feedback.ts:16]: update the "lit-html" to latest  "lit"

**Review by gisumit — CHANGES_REQUESTED** [review]

**Comment by gisumit** [comment]
can you add the screenshot for the firefox browser.

File: packages/opc-feedback/src/opc-feedback.ts (hunk 1 above)
Comment (riginoommen) [reply packages/opc-feedback/src/opc-feedback.ts:16]: Updated

Comment (gisumit) [inline packages/opc-feedback/src/opc-feedback.ts:16]:
```suggestion
import { repeat } from 'lit/directives/repeat.js';
```

**Review by gisumit — CHANGES_REQUESTED** [review]

File: packages/opc-feedback/src/opc-feedback.ts (hunk 1 above)
Comment (riginoommen) [reply packages/opc-feedback/src/opc-feedback.ts]: Changed

Comment (gisumit) [reply packages/opc-feedback/src/opc-feedback.ts]: @internalProperty() is changed to @state

Screenshot 2021-07-13 at 12 44 22 PM

Comment (gisumit) [reply packages/opc-feedback/src/opc-feedback.ts]: @riginoommen can you change the `@internalProperty` to `@state`

**Review by hybridx — APPROVED** [review]
LGTM

**Review by hybridx — APPROVED** [review]

**Review by gisumit — APPROVED** [review]

**Pull request merged**